run-ci.sh

```diff
@@ -29,6 +29,10 @@ go mod download
 go build ./pulsar
 go build -o pulsar-perf ./perf
 
+# Project code style check
+wget -O - -q https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s v1.18.0
```

Comment:
This should go in the Docker image so that we can cache it

Reply:
Good idea @merlimat, has been fixed, please take a look again, thanks.